aten/src/ATen/templates/RedispatchFunctions.cpp

```diff
@@ -0,0 +1,14 @@
+// ${generated_comment}
```

Comment:
I split off the redispatch fn definitions into this new file to avoid making `Functions.cpp` enormous.

I'm not sure if I can do this though... my builds are all failing with missing symbols, so the linker isn't picking up my redispatch definitions here. Am I missing something? I looked for all of the places that similar files (`Functions.cpp` and `TensorMethods.cpp`) are referenced in build scripts, and only found one location, `BUILD.bazel`.

I could dump everything back into `Functions.cpp`, but that bumps up the size from ~18k to ~36k lines.

Reply:
CI doesn't seem to be complaining. But either way, you're gonna need to tell some areas of the build system about RedispatchFunctions.cpp (I think cmake will pick it automatically but pretty sure buck won't). You can find a recent PR that rejiggered this stuff, or grep for Functions.cpp and find all the sites its referenced in fbcode.